middleware/jwt/authentification.js

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+import keys from './keys';
+
+const generateToken = payload => `bearer ${jwt.sign(payload, keys.secret, { expiresIn: '1h' })}`;
```

Comment:
I would suggest making this function async, and await jwt.sign() to avoid any asynchronous errors.

Reply:
I think the jwt.sign is synchrone ;
https://www.npmjs.com/package/jsonwebtoken#jwtsignpayload-secretorprivatekey-options-callback

Reply:
Alright, @Karlmusingo .